tests/integration/tests/conftest.py

```diff
@@ -86,9 +88,32 @@ def instances(
         instances.append(instance)
         util.setup_k8s_snap(instance, snap_path)
 
-    first_node, *_ = instances
-    first_node.exec(["k8s", "bootstrap"])
+    # first_node, *_ = instances
+    # first_node.exec(["k8s", "bootstrap"])
```

Comment:
so?  Who bootstraps the cluster now?  I only see a bootstrap now in `test_etcd`.  Where do the other tests get their k8s cluster bootstrapped?